CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,333 @@
+events and processes taking place within the **Open Research Community Building Team**. 
+Specifically, our Code of Conduct applies to any conduct:
+- in interactions within the team/community
+- at any team activity, communication or event
+- outside the team activity when a team member is representing projects associated with the team
+- by members outside the team engaging on the Open Community Building GitHub repository.
+
+**10.2** This Code of Conduct also applies to
+individuals officially representing the community in public spaces.
+
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+a representative at an online or offline event.
+
+**10.3** This Code of Conduct also applies to individuals engaging with this repository 
+
+for instance, in interactions on GitHub issues and Pull Requests or activities taking 
+place in the context of the **Open Research Community Building Team**.
+
+## 11. Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behaviour and will take appropriate and fair corrective, restorative and 
+action in response to any behaviour that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## 12. Enforcement
+
+Instances of trolling, harassing, abusive, or otherwise unacceptable behaviour should be
+reported to the community leaders responsible for enforcement.
+If noticed by community leaders, incidents should be recorded and shared with one or more 
+members of the Open Research Community Building Team for transparency.
+
+All reports will be reviewed and investigated promptly and fairly.
+
+All community leaders are obligated to respect the privacy and security of the
+reporter of any incident.
+Together we strive to foster an environment where everyone feels valued, respected, and included.
+
+## 13. Reporting Incident
+
+**13.1** Code of Conduct Contact for Reporting
+
+If you experience or witness unacceptable behaviour, or have any other concerns, please report it 
+by contacting the Team Lead, Malvika Sharan (email: msharan@turing.ac.uk). Malvika may involve
+the Tools, Practices and Systems Programme Director, Kirstie Whitaker when incidents are perceived 
+as difficult (not straightforward) except when any potential conflict of interest may arise.
+
+To report an issue involving the team lead, please email directly to the Tools, Practices and Systems 
+Programme Director, Kirstie Whitaker. Since she is currently on maternity leave, please contact TPS
+Programme Manager Arielle Bennett (abennett@turing.ac.uk) - who is also our ombudsperson. 
```

Comment:
@arielle-bennett thanks for agreeing to be added in the CoC. Can you please look at the CoC and the details that this role entails?

Reply:
Approved by Arielle personally.